content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -96,22 +115,68 @@ If you haven't already set up a cluster locally, run `minikube start` to create
    web       NodePort   10.104.133.249   <none>        8080:31637/TCP   12m
    ```
 
-1. Visit the Service via NodePort:
+1. Visit the Service via NodePort, using the [`minikube service`](https://minikube.sigs.k8s.io/docs/handbook/accessing/#using-minikube-service-with-tunnel) command. Follow the instructions for your platform:
 
+   {{< tabs name="minikube_service" >}}
```

Comment:
Although the PR [description](https://github.com/kubernetes/website/pull/45071#issue-2126466789) mentions that Windows instructions haven't been tested yet, I'd like to highlight a minor observation looking at the [preview page](https://deploy-preview-45071--kubernetes-io-main-staging.netlify.app/docs/tasks/access-application-cluster/ingress-minikube/) both the MacOS and Windows tabs seem to have similar instructions for now, which gives an impression that the data doesn't load properly in the tab when clicking on the tab name. Perhaps we could streamline this by combining them and labeling the tab as "MacOS or Windows" if the instructions are indeed the same.

Reply:
I agree. I after testing it is observed that the Windows instructions and output are similar to the Mac, then we can combine the tabs. I need a person who is willing to collaborate with me on this PR by testing the steps on Windows. I'll send out a request on slack.

Reply:
@dipesh-rawat I'm unable to find a contributor who can test the instructions on a Windows machine. On the other hand, readers on MacBook machines frequently fail on this tutorial.

This PR fixes issues https://github.com/kubernetes/website/issues/44651 and https://github.com/kubernetes/website/issues/45027 and both are raised for MacBook.

Is it fine if we reduce the scope of this PR to fix the instructions for MacBook alone (as that is the need of the hour)?
The changes for Windows can be tackled in another PR if the need arises.

Let me know your view. If you agree, I will drop the Windows related changes and submit a new commit.

Reply:
I issues a new comment to the thread to follow-up on this point.